dotnet/targets/Xamarin.Shared.Sdk.targets

```diff
@@ -973,7 +973,7 @@
 			<!-- include .dll, .exe and, for debugging only, .pdb files inside the .app -->
 			<ResolvedFileToPublish
 				Update="@(ResolvedFileToPublish)"
-				RelativePath="$(_AssemblyPublishDir)\%(ResolvedFileToPublish.DestinationSubDirectory)\%(Filename)%(Extension)"
+				RelativePath="$(_AssemblyPublishDir)\%(ResolvedFileToPublish.RelativePath)"
```

Comment:
c.c. @rolfbjarne 

https://github.com/xamarin/xamarin-macios/pull/12333/files#diff-2e24f1b6f44df7580244d00f106fa1a9e2257536fbca9d7bbdc74571549324a9L952

I'm surprised this did not trigger any of our existing tests.

Reply:
Yes, this has been discussed a bit here: https://github.com/xamarin/xamarin-macios/issues/12386#issuecomment-898562065

For normal project references our logic works just fine, but the test case is setting `CopyToOutputDirectory=PreserveNewest` on `None` items, which is something we don't have tests for.